pages/entropy/best-practices.mdx

```diff
@@ -1,5 +1,22 @@
 # Best Practices
 
+## Limit gas usage on the callback
+
+Since the entropy providers limit the gas usage on the callbacks,
+it is important to keep the callback function as simple as possible
+so that the gas usage is predictable and below the limits.
+This will ensure consistent performance and avoid potential issues with the callback.
```

Comment:
This first paragraph can be rephrased like 

`Keeping the callback function simple is crucial because the entropy providers limit gas usage. 
This ensures gas usage is predictable and consistent, avoiding potential issues with the callback.`